### PR TITLE
Add information about open in form

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add information about open in form functionality. See the related PRs for more context.

#### Related PRs
https://github.com/mattermost/mattermost-webapp/pull/8597
Mobile: TBD

### CODE REVIEW

File: site/content/integrate/apps/interactivity/_index.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ On submit, the submit call to the call endpoint will be sent. The submit call wi
 
 Commands arguments are treated as forms. When a leaf command is typed, the arguments of the command are fetched. If the command binding has a form attached, those will be used. If not, a form call will be made to the command call. The call will include in the context the app ID, user ID, the post ID, the root post ID (if any), the channel ID, and the team ID. The call will expect a form response.
 
+During autocomplete, the user can open the form as a Modal Form to finish completing the command. Any fields not supported by commands (like markdown fields) or form attributes not visible in commands (like the title) will be shown when opened as a Modal Form.
```
Comment (justinegeffen) [inline site/content/integrate/apps/interactivity/_index.md:92]:
```suggestion
During autocomplete, the user can open the form as a Modal form to finish completing the command. Any fields not supported by commands (like markdown fields) or form attributes not visible in commands (like the title) will be shown when opened as a Modal form.
```

**Review by justinegeffen — APPROVED** [review]
Two corrections but otherwise good to go! :)

**Review by hanzei — COMMENTED** [review]
LGTM

**Review by hanzei — COMMENTED** [review]
LGTM

**Review by hanzei — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Pull request merged**